### ISSUE TITLE
Unable to include abstract-sdk@0.7.4

### ISSUE THREAD

**Issue by gerrit** [goabstract/abstract-sdk#102]
When including `abstract-sdk@0.7.4` (failed to install a more recent version as per #101)

Seems like abstract has undeclared dependencies on `core-js` and `regenerator-runtime` packages. But even when manually installing these, the simplest-possible inclusion of the abstract sdk fails with a `ReferenceError`.

Abstract.app Version 77.0.1 (77.0.1.12639)
node v11.10.1
macOS 10.14.3 (18D109)

## Steps to reproduyce:

`index.js`:
```
const Abstract = require("abstract-sdk");
```

```
> yarn add abstract-sdk@0.7.4
[…]
> node index.js
internal/modules/cjs/loader.js:615
    throw err;
    ^

Error: Cannot find module 'core-js/modules/es6.object.define-property'
[…]

> yarn add core-js
[…]
> node index.js

internal/modules/cjs/loader.js:615
    throw err;
    ^

Error: Cannot find module 'regenerator-runtime/runtime'
[…]

> yarn add regenerator-runtime
[…]
> node index.js
/Users/gerrit/spotify/abstracttest/node_modules/abstract-sdk/lib/AbstractCLI/index.js:576
      regeneratorRuntime.mark(function _callee14(args) {
      ^

ReferenceError: regeneratorRuntime is not defined
    at /Users/gerrit/spotify/abstracttest/node_modules/abstract-sdk/lib/AbstractCLI/index.js:576:7
    at /Users/gerrit/spotify/abstracttest/node_modules/abstract-sdk/lib/AbstractCLI/index.js:646:6
    at Object.<anonymous> (/Users/gerrit/spotify/abstracttest/node_modules/abstract-sdk/lib/AbstractCLI/index.js:698:2)
    at Module._compile (internal/modules/cjs/loader.js:738:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:749:10)
    at Module.load (internal/modules/cjs/loader.js:630:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:570:12)
    at Function.Module._load (internal/modules/cjs/loader.js:562:3)
    at Module.require (internal/modules/cjs/loader.js:667:17)
    at require (internal/modules/cjs/helpers.js:20:18)
```

**Comment by gerrit** [comment]
Update: `0.7.3` works fine after manually adding `core-js` dependency

**Comment by bitpshr** [comment]
Thanks for taking the time to file this and #101 @gerrit. This dependency issue should be fixed in `abstract-sdk@0.7.5`, which is now available in the public NPM registry. Please don't hesitate to let us know if you have any additional issues while testing and using the SDK.

**Comment by gerrit** [comment]
Appreciate the reactiveness and super-quick fixes here! 🙇
